src/renderers/dom/shared/__tests__/ReactDOMServerIntegration-test.js

```diff
@@ -432,9 +432,9 @@ describe('ReactDOMServerIntegration', () => {
     });
 
     describe('unknown attributes', function() {
-      itRenders('no unknown attributes', async render => {
-        const e = await render(<div foo="bar" />, 1);
-        expect(e.getAttribute('foo')).toBe(null);
+      itRenders('unknown attributes', async render => {
+        const e = await render(<div foo="bar" />, 0);
```

Comment:
Tiny nit: you should be able to just get rid of the second argument here and on line 448 below; the error count argument defaults to 0.

Nice job!